apps/api/src/app/integrations/usecases/create-integration/create-integration.usecase.ts

```diff
@@ -149,7 +149,7 @@ export class CreateIntegration {
       const defaultName =
         providers.find((provider) => provider.id === command.providerId)?.displayName ?? providerIdCapitalized;
       const name = command.name ?? defaultName;
-      const identifier = command.identifier ?? `${slugify(name)}-${shortid.generate()}`;
+      const identifier = command.identifier ?? `${slugify(name)}-${shortId()}`;
```

Comment:
before, we used shortId, which generated ids [7+ characters long](https://github.com/dylang/shortid/blob/ef422f710396ef5b637bcf673772fa3b0bf8e3ef/README.md?plain=1#L14)

do we want to increase the length here as well?  
reasons:  
1. to align with old ids.  
2. for future multi-tenancy support, which may require a larger identifier.

Reply:
Good point, I increased the length to 8.